### ISSUE TITLE
flutter_style_todos, allow multiple leading `// `

### ISSUE THREAD

**Issue by scheglov** [dart-lang/linter#4803]
When we comment out code with existing `// TODO(name): text`, it becomes
```dart
//      // TODO(name): text
//      print(42);
```

This code reports the lint.
I propose to allow multiple `// `  or `/// ` prefixes, or ignore multi-level single line comments in the lint.

**Comment by pq** [comment]
@hixie,  @goderbauer: thoughts? Any other flutter folks who may want to weigh in?

**Comment by goderbauer** [comment]
+100 :)

I occasionally run into the same problem and wish it wouldn't report a lint there. Never got around to file a bug for it, though.

**Comment by Hixie** [comment]
i don't much care about this particular case since we'd never want that kind of code checked in anyway :-)
